### ISSUE TITLE
Whitgift School Planning 2022-23

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMycetoma/Series-2-Aminothiazoles#12]
Further to the [general idea](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/83#issuecomment-1325735140), and [the current needs of the series](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/blob/master/File%20Diary/2022%20Whitgift%20School%20Involvement/Deck%20of%20Route%20and%20Targets.pptx), Whitgift School in the UK are interested in contributing a molecule.

Which molecule to make, and how to get it?

The obvious idea is to change the red aniline in the synthetic scheme i.e. the school needs to buy an aniline that is different from everything else that has been/is being made and keep the other reagents the same.

I think @kym834 was suggesting modifying the ketone with:

Name | Link | Cost per g (£)
-- | -- | --
2-Bromo-3'-chloropropiophenone | https://www.thermofisher.com/order/catalog/product/H64999.06?SID=srch-srp-H64999.06 | 14.42
2-Bromo-2'-nitroacetophenone | https://www.thermofisher.com/order/catalog/product/398850050?SID=srch-srp-398850050 | 20.8
5'-Bromo-2'-hydroxyacetophenone | https://www.thermofisher.com/order/catalog/product/H61979.09?SID=srch-srp-H61979.18 | 3.19

or

2-Bromo-1-(2-quinolinyl)ethanone

2-Bromo-1-(5-chloro-2-pyridinyl)ethanone

2-Bromo-1-(3,5-dichloro-2-pyridinyl)ethenone

But I think that the school is having trouble ordering these. @kym834 list of suggestions is [here](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/blob/master/File%20Diary/2022%20Whitgift%20School%20Involvement/Kymberley's%20suggested%20list.pdf).

They have ordered:

Benzoyl chloride

2-amino-4-methylpyridine

2-Bromo-4-nitroacetophenone

I guess we need to be clear on what the target is, and that they can order the relevant materials. I've asked someone in the school to sign up to Github so we can discuss all this here.

I have set up a lab notebook for the students to use [here](https://uk-mynotebook.labarchives.com/share/Whitgift%2520School%2520MycetOS/MC4wfDE5NjE0LzAvVHJlZU5vZGUvNDQ3MjQzMjMzfDAuMA==).

**Comment by kym834** [comment]
Yes, the suggestions I made from a quick SciFinder search seem to be hard for the school to get. I don't know if this would be different for a school based say in the USA... or if schools are limited to who they purchase from. The three in the table are compounds Whitgift found so I suggested going with one of them as it is easy for them to get hold of it.

I went with changing the a-bromoketone only because I was waiting for the results from the latest shipment of samples that changed the aniline (which we now have!) to see if there was anything good in there. No such luck it seems, so exploring this more is definitely needed.

@mattodd

**Comment by mattodd** [comment]
Hi @kym834 - maybe, but those kinds of lists can often go out of date and people will always reach out for advice. In this case I'm happy to suggest something general and we can see whether the school can order suitable reagents, or we order at UCL and send over. Would you be able to link to the latest data in this series so we can have a think about what's missing? You don't have any of the intermediate thiourea kicking around by any chance, do you?

**Comment by kym834** [comment]
@mattodd, yeah true. I've found the bromoketones hard to source, especially with the pyridine ring. It might be that we want to brominate so ketones and have them ready to ship to schools who are interested and can't get hold of any suitable reagents. Obviously work that I can't do :( so maybe to grand of an idea.

I'm working on updating the data today so hopefully that will be up soon. 

Yes... I think there should be some at USyd. @KlementineJBS or @fantasy121 could you please check and let us know?

**Comment by MFernflower** [comment]
![image](https://user-images.githubusercontent.com/3164942/203676287-700143a0-950c-4d38-ba0e-4cc2e24c89f2.png)


@kym834 @mattodd some dearomatized ideas

**Comment by mattodd** [comment]
OK, so the results are in! @OpenSourceMycetoma/corecontrib 

Whitgift school made _N_-(4-methylpyridin-2-yl)-4-(4-nitrophenyl)thiazol-2-amine. 

![What Did Whitgift Make](https://user-images.githubusercontent.com/4386101/229436990-c1e82878-404e-4ff3-9279-f1f69b86cb6d.png)

CC1=CC(NC2=NC(C3=CC=C([N+]([O-])=O)C=C3)=CS2)=NC=C1
InChI=1S/C15H12N4O2S/c1-10-6-7-16-14(8-10)18-15-17-13(9-22-15)11-2-4-12(5-3-11)19(20)21/h2-9H,1H3,(H,16,17,18)
ELNs are here: https://tinyurl.com/WhitgiftSchoolELN

NMRs:
[1D.pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11135742/1D.pdf)
[1D-2.pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11135743/1D-2.pdf)
[1D-3.pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11135746/1D-3.pdf)

Anyone want to comment on whether these NMRs support the structure? ;)

The LCMS trace supports MH+ 313.

@dmitrij176 Can you please share here the raw NMR data files (the above are just PDFs) and the LCMS trace that you took? Chris Waudby has the NMR files. If you could please also analyse the other Whitgift samples - just 1H in DMSO and LCMS for now, we can then decide the "team winner based on purity" for the school.

@kym834 we need to get together with @JuliaSarju and @AndyBishopSevenoaks and all others to plan what happens after Easter. Any time April 13th/14th/17th/18th?

Here are the Whitgift students, triumphant.

![20230330_144336 Small](https://user-images.githubusercontent.com/4386101/229439009-c28c978a-b51d-4cc6-b39c-93e7ddad210c.jpg)

**Comment by JuliaSarju** [comment]
Congratulations! We've had a successful time at York too. The students are analyzing their results and making a poster. 

Meeting wise I could do the 14th or 17th April

Julia

**Comment by kym834** [comment]
Excellent news! Both for Whitgift and York. I know that Sevenoaks have got started as well so hopefully they are also successful :)

10 PM on 14th or 17th of April works best for me. Time conversion below - we'd actually all be living the same day haha
![image](https://user-images.githubusercontent.com/61669512/229658067-0291db2a-264d-45c2-a912-eca194c56b70.png)

**Comment by dmitrij176** [comment]
Good day. 
Please find attached the characterisation data results:
**LCMS**
[Wh.S (SS).pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11220978/Wh.S.SS.pdf)
[WS (Arren).pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11220979/WS.Arren.pdf)
[WS (PS).pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11220980/WS.PS.pdf)
[WS (RLPA).pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11220981/WS.RLPA.pdf)
[WS (TS).pdf](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11220982/WS.TS.pdf)

**Comment by dmitrij176** [comment]
**1H NMR**
[WS (Arren)-Apr06-2023-11.zip](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11221006/WS.Arren.-Apr06-2023-11.zip)
[WS (PS) -Apr06-2023-12.zip](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11221007/WS.PS.-Apr06-2023-12.zip)
[WS (RLPA) -Apr06-2023-14.zip](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11221008/WS.RLPA.-Apr06-2023-14.zip)
[WS (SS)-Apr06-2023-15.zip](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11221009/WS.SS.-Apr06-2023-15.zip)
[WS (TH) -Apr06-2023-13.zip](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/11221010/WS.TH.-Apr06-2023-13.zip)

**Comment by dmitrij176** [comment]
The cleanest samples which have predominantly product peak at m/z 313 are:
WS (SS)
WS (PS)
WS (TH)
The other two samples probably have starting materials left, but in general all crude mixtures contain the desired organic product.

**Comment by kym834** [comment]
@mattodd, @JuliaSarju @AndyBishopSevenoaks 
Are we planning to meet either Today or on Monday?

**Comment by mattodd** [comment]
Hi @JuliaSarju @AndyBishopSevenoaks @kym834 @Whitgift - sorry we failed to find a time. Let's try again, since Whitgift are keen to get going on the next target. How about some time April 26th/27th, e.g. 11 a.m. or noon one of those days, which is 8pm or 9pm Sydney time? Any good for everyone? Would be great to lock it in and share results/plans.

**Comment by kym834** [comment]
@mattodd, @JuliaSarju , @AndyBishopSevenoaks, @Whitgift either of those days at that time is OK for me.

**Comment by AndreaOtago** [comment]
If it's not too late in NZ time I might join (we have some DNDi/OSN benzoxazole synthesis about to start) but don't arrange the time to suit me :)

**Comment by PaulMcKeatingSevenoaks** [comment]
Hi - any of those times on 26th or 27th suit me. We hopefully have the last stage of our synthesis tomorrow (20th).

**Comment by PaulMcKeatingSevenoaks** [comment]
Hi - any of those times on 26th or 27th suit me. We hopefully have the last stage of our synthesis tomorrow (20th).
@mattodd @JuliaSarju @Whitgift @kym834

**Comment by JuliaSarju** [comment]
Hi everyone!

I could do 11-2 on Wednesday 26th (UK) but I'm not available on the 27th.
It would be great to catch up!

Julia
Dr Julia Sarju FHEA , [image: A button with "Hear my name" text for name
playback in email signature] <https://www.name-coach.com/julia-sarju>
She/Her
Lecturer in Chemistry, Year One Leader, and Chemistry Disability Contact
The University of York

*UG + PG Chemistry Students:* Do you have experiences of inequalities in
chemistry education that you want to share with chemistry instructors? Are
there actions you would like chemistry departments to take? Add your voice
and take part in this RSC Diversity and Inclusion project: Listening to
Inequalities in Chemistry Education
<https://docs.google.com/forms/d/e/1FAIpQLSfF-KSH-VXQOb55kLV_URwZkZVHB32QVM5TdapgSmEtJ3nTWQ/viewform?usp=sf_link>

Join the Student Disability Forum Mailing List
<https://docs.google.com/forms/d/e/1FAIpQLSejLDEYk84XxYXmFQmWphP0At3d0rjbIw_xrgVwS9hOAK7zRw/viewform>

**Comment by mattodd** [comment]
OK all @PaulMcKeatingSevenoaks @JuliaSarju @Whitgift @kym834 please pencil in Wed April 26th at 12 noon UK. Am just confirming with Whitgift.
When we're locked in we can plan the meeting to be sure we're up to date with where we are and what we need. It would be VERY helpful if people can prepare a slide to share that shows where you're up to. @kym834 please reach out to others in Breaking Good - Barker College was it? - to ask them to come along if interested. We can also tweet out in case other schools want to see how this all works for the future.

**Comment by kym834** [comment]
I've got in touch with @KlementineJBS to give us an update on what SSP and the high school workshops at USyd are planning for this year and I'll give Barker a buzz too and see if they can't make it as well.

**Comment by mattodd** [comment]
A reminder of the meeting Wed 26/4/23 at 12 noon, at https://ucl.zoom.us/j/99789829423. Have also tweeted out in case other interested parties would like to see where we're up to. 
https://twitter.com/MatToddChem/status/1650981843339034625

**Comment by kym834** [comment]
The next molecule Whitgift will be making is

![2023_02_Whitgift_School](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/assets/61669512/3599ca59-b6b5-4700-a018-4a3e02a0aae6)

They'll be starting this in September 2023.

**Comment by KlementineJBS** [comment]
@Whitgift hiya - I'm trying to update our masterlist. Do you know if this compound has been given a MYOS code? And do you refer to it with any other sample id yourself? (i.e. Whitgift_2023)

And how's the synthesis going?

**Comment by eloizidou** [comment]
Hi, these are the NMR spectra for the Whitgift molecule, that we analysed in March.
[Whitgift_NMR_sample1.docx](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/files/15133148/Whitgift_NMR_sample1.docx)
![whitgift_1](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/assets/146206920/c8837d51-8be5-43d8-82e9-7aaa792042f7)
The second molecule that Whitgift made is shown below and this will be analysed in June.
![whitgift_2](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/assets/146206920/8e444258-beb0-4e93-b60b-6fec72dc3f92)

**Comment by KlementineJBS** [comment]
Hi Erika - do you mean analysed as in characterised/tested for purity? 
I'll make sure this is updated in the SAR page for Series 2.

**Comment by eloizidou** [comment]
Hi Klementine, yes analysed for purity. I don't think these two Whitgift molecules have been analysed for activity yet.

Sent from Samsung Mobile on O2
Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by KlementineJBS** [comment]
Have you assigned MYOS codes to either of these compounds, or would you like me to do that?

**Comment by eloizidou** [comment]
Hi Klementine, I haven't assigned a MYOS code yet, could you please do it? Many thanks.

Sent from Samsung Mobile on O2
Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by KlementineJBS** [comment]
Turns out one already had a MYOS code - they are now MYOS432 (the one with a methyl) and MYOS605. I've also updated the SAR more generally [here](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/wiki/Overall-summary).
